src/index.js

```diff
@@ -17,8 +17,9 @@ function capitalizeFirstLetter(string) {
 function checkEndpoint(url, endpoints) {
 	const parser = window.document.createElement('a');
 	parser.href = url;
+	const host = parser.host ? parser.host : window.location.host;
```

Comment:
Totally true, it's quite a workaround (although just of IE btw). 
But if the url is relative, it's always relative to the `window.location.host`? What about verifying that the url is relative? 

[url-parse](https://github.com/unshiftio/url-parse) also seems to use this `window.location.host`:

> baseURL (Object | String): An object or string representing the base URL to use in case url is a relative URL. This argument is optional and defaults to location in the browser.